inc/patterns/query-simple-blog.php

```diff
@@ -6,7 +6,7 @@
 	'title'      => __( 'Simple blog posts', 'twentytwentytwo' ),
 	'categories' => array( 'query' ),
 	'blockTypes' => array( 'core/query' ),
-	'content'    => '<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true,"perPage":10},"layout":{"inherit":true}} -->
+	'content'    => '<!-- wp:query {"query":{"perPage":1,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"layout":{"inherit":true}} -->
```

Comment:
Should we be consistent with `core/GB` and the rest of the patterns here about the `inherit` value of the `Query Loop` block?

Reply:
Yeah, I think we should. It's an oversight that it's not doing that currently.

Reply:
I updated it - it was one instance and also made some updates in the number for page patterns.